llm/llm.go

```diff
@@ -79,6 +79,24 @@ func New(workDir, model string, adapters, projectors []string, opts api.Options)
 			break
 		}
 
+		if info.Library == "tegra" {
```

Comment:
I'm not seeing where this is set.  Maybe I'm missing it, but is this stale code perhaps?

Reply:
I forgot that this is still in here. Originally it just automatically set NumGPU to 999 for tegra devices if not manually set in env variable. I'll remove it